spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -127,8 +127,8 @@
       end
     end
 
-    describe "personal details if the jobseeker has a blank previous job application" do
-      let!(:previous_application) { create(:job_application, :status_draft, jobseeker:, first_name: nil, last_name: nil, phone_number: "01234567890") }
+    describe "personal details if the jobseeker has a blank previously submitted job application" do
+      let!(:previous_application) { create(:job_application, :status_submitted, jobseeker:, first_name: nil, last_name: nil, phone_number: "01234567890") }
```

Comment:
A submitted application without data? How is this achievable? It seems a non-reachable context.